src/test/error-pages.test.ts

```diff
@@ -1,14 +1,26 @@
+jest.mock("ioredis");
+
 import request from "supertest";
+import { CsrfError } from "@companieshouse/web-security-node";
 
 import app from "../app";
-import * as config from "../config";
-
 import { get } from "../controllers/start.controller";
+import * as config from "../config";
 
 jest.mock("../controllers/start.controller");
+
 const mockGet = get as jest.Mock;
 
+const CSRF_TOKEN_ERROR = "CSRF token mismatch";
```

Comment:
if a value is only used once, there is no point in creating a constant, it is better to use it directly

Reply:
The constants help to make the test more readable and make it consistent with the layout of other tests, so unless there's a real functional need to change this, I would say we should keep it as it is.

Reply:
I can't see what makes the test more readable.
A value can be stored in a variable if it is used in at least two different places.

Reply:
As this seems to be a problem I have changed it to use the values inline.

Reply:
@lduranteau did you want to add anything further if not can we resolve this?